fastlane_core/spec/cert_checker_spec.rb

```diff
@@ -112,10 +112,12 @@ class ProcessStatusMock
           `ls`
 
           keychain = "keychain with spaces.keychain"
-          cmd = %r{curl -f -o (([A-Z]\:)?\/.+) https://www\.apple\.com/certificateauthority/AppleWWDRCAG6\.cer && security import \1 -k #{Regexp.escape(keychain.shellescape)}}
+          check_cmd = %r{curl -f -o (([A-Z]\:)?\/.+) https://www\.apple\.com/certificateauthority/AppleWWDRCAG6\.cer && security verify-cert -c \1}
+          import_cmd = "security import /AppleWWDRCAG6/i -k #{Regexp.escape(keychain.shellescape)}"
           require "open3"
 
-          expect(Open3).to receive(:capture3).with(cmd).and_return(["", "", success_status])
+          expect(Open3).to receive(:capture3).with(check_cmd).and_return(["...certificate verification successful.", "", success_status])
+          expect(Open3).to receive(:capture3).with(import_cmd).and_return(["", "", success_status])
```

Comment:
FYI, If you decide to accept my suggestions above and go with passing arguments to `Open3.capture3` separately, this could then look like:
```ruby
expect(Open3).to receive(:capture3).with('curl', '-f', '-o', anything, 'https://www.apple.com/certificateauthority/AppleWWDRCAG6.cer')
expect(Open3).to receive(:capture3).with('security', 'verify-cert', '-c', anything)
expect(Open3).to receive(:capture3).with('security', 'import', anything, anything)
```
Note: [`anything` is a special matcher in RSpec](https://rubydoc.info/gems/rspec-mocks/RSpec/Mocks/ArgumentMatchers) to mean that it can expect an argument with any value at this position.